docs/mp/websocket.adoc

```diff
@@ -174,6 +174,47 @@ All endpoint methods in Helidon MP are executed in a separate thread pool, indep
 
 For more information see the link:{helidon-github-tree-url}/examples/webserver/websocket[example].
 
+=== WebSocket Endpoints on Different Ports
+
+The Helidon Reactive Webserver can listen on multiple ports or sockets. This can be useful when APIs for different type of users need to be exposed (such as admin vs. non-admin users). Just like for REST resources, it is possible to expose  WebSocket applications on different ports _provided that the routing paths are different_ --this is due to a constraint in Tyrus, given that it is simply unaware that the endpoints are bound to different ports in the Helidon Webserver.
```

Comment:
In the docs we are referring to it as Reactive Web**S**erver or Helidon Web**S**erver. When referring to a generic web server it's two words, lowercase web server.

https://helidon.io/docs/v3/#/se/webserver

Reply:
OK, I will change that, but the section title on the left panel reads "Reactive Webserver" as written above.